src/Tribe/Views/V2/Assets.php

```diff
@@ -99,6 +99,49 @@ public function register() {
 			]
 		);
 
+		/**
+		 * @todo: @paulmskim figure out how to enqueue only when widget is available.
+		 * @todo: @paulmskim commented out for release so widget styles do not enqueue, uncomment when actually releasing.
+		 */
+		// tribe_asset(
+		// 	$plugin,
+		// 	'tribe-events-widgets-v2-skeleton',
+		// 	'widgets-skeleton.css',
+		// 	[
+		// 		'tribe-common-skeleton-style',
+		// 	],
+		// 	'wp_enqueue_scripts',
+		// 	[
+		// 		'priority'     => 10,
+		// 		'conditionals' => [ $this, 'should_enqueue_frontend' ],
+		// 		'groups'       => [ static::$group_key ],
+		// 	]
+		// );
+
+		/**
+		 * @todo: @paulmskim figure out how to enqueue only when widget is available.
+		 * @todo: @paulmskim commented out for release so widget styles do not enqueue, uncomment when actually releasing.
+		 */
+		// tribe_asset(
```

Comment:
:warning: This comment is 50% valid code; is this commented out code?

<sub>hash: 8aa9dbf4c60427a36fd28876dba5f0da</sub>